runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/internal/runtime/InternalPlatform.java

```diff
@@ -803,13 +737,8 @@ private void closeOSGITrackers() {
 	 * Pre-pend the message with the current date and the name of the current thread.
 	 */
 	public static void message(String message) {
-		StringBuilder buffer = new StringBuilder();
-		buffer.append(new Date(System.currentTimeMillis()));
-		buffer.append(" - ["); //$NON-NLS-1$
-		buffer.append(Thread.currentThread().getName());
-		buffer.append("] "); //$NON-NLS-1$
-		buffer.append(message);
-		System.out.println(buffer.toString());
+		System.out.println(
```

Comment:
Using String.format would make this more readable

Reply:
Personally I find String.format more readable.
I understand and agree that splitting the string with `+arg+` also has its difficulties to read it, but 'manually' identifing the markers in the format-String and matching it to the appended args is also something I don't consider very readable.
And unfortunately [String templates](https://openjdk.org/jeps/430) are not (yet) available in Java.

Reply:
For me its hard to understand how the final message is supposed to looks like. Having placeholders is very common concept for example in SLF4J API, also eclipses `//$NON-NLS-1$` don't make it better here

Reply:
> For me its hard to understand how the final message is supposed to looks like. Having placeholders is very common concept for example in SLF4J API, also eclipses `//$NON-NLS-1$` don't make it better here

Yes, that's all true, but at least if there are many arguments I personally find that approach not very readable as well.
Anyways, since there are only three arguments I agree with you and changed it.